script.sh

```diff
@@ -208,14 +210,45 @@ RUN echo "deb-src ${TRAVIS_DEBIAN_MIRROR} experimental main" >> /etc/apt/sources
 EOF
 fi
 
+EXTRA_PACKAGES=""
+
+case "${TRAVIS_DEBIAN_EXTRA_REPOSITORY}" in
+	https:*)
+		EXTRA_PACKAGES="apt-transport-https"
```

Comment:
Right, but then you need to do ``EXTRA_PACKAGES="${EXTRA_PACKAGES} apt-transport-https"``  :)